src/features/core/deviceModeTransformation/transformationHandler.js

```diff
@@ -83,19 +83,8 @@ class TransformationsHandler {
                       }]
                     } 
                 */
-                /**
-                 * Filter the successful transformed event
-                 */
-                const transformationResponse = [];
-                response.transformedBatch.forEach((dest) => {
-                  transformationResponse.push({
-                    id: dest.id,
-                    payload: dest.payload.filter((tEvent) => tEvent.status === '200'),
```

Comment:
why we took this one off? according to DMT developers we should be getting errored response objects too in the array. should we not sent only 200 status with not null | undefined value for event property?

Reply:
I have moved the code from here to analytics.js so the we can better handle the error message.